src/apps/expenses/components/CreateExpenseForm.js

```diff
@@ -504,15 +504,15 @@ class CreateExpenseForm extends React.Component {
   }
 
   render() {
-    const { LoggedInUser, collective } = this.props;
+    const { LoggedInUser } = this.props;
 
     if (!LoggedInUser) {
       return (
         <div className="CreateExpenseForm">
-          <P textAlign="center" mt={4} fontSize="LeadParagraph" lineHeight="LeadParagraph">
-            <FormattedMessage id="expenses.create.login" defaultMessage="Sign up or login to submit an expense." />
+          <P textAlign="center" mt={4} fontSize="LeadParagraph" lineHeight="LeadParagraph" marginbottom="40">
```

Comment:
`marginbottom="40"` is not a valid prop. See https://styled-system.com/api#space